examples/3d/3d_scene.rs

```diff
@@ -40,3 +46,27 @@ fn setup(
             ..Default::default()
         });
 }
+
+fn follow(
+    mut q: Query<&mut Transform, With<Follow>>,
+    q_camera: Query<(&Camera, &GlobalTransform)>,
+    windows: Res<Windows>,
+    mut evr_cursor: EventReader<CursorMoved>,
+) {
+    // Assumes there is at least one camera
+    let (camera, camera_transform) = q_camera.iter().next().unwrap();
```

Comment:
This should use the nice `.single()` helper function, designed for exactly this purpose.

Reply:
The code *should* fail and need to be rewritten if there's more than one camera returned.